### PR TITLE
Consolidate workflow jobs; test library with 1.46 MSRV and CLI with 1.54

### PR DESCRIPTION
**Pull request by bgilbert**
See individual commit comments and discussion in #77.

The README badge would be more clearly correct if we didn't enable the `cli` feature by default, but that might be too much of a breaking change.

### CODE REVIEW

**Comment by cecton** [comment]
No that's fine. If I release 1.0 it's a major so we can afford a breaking change.

**Comment by bgilbert** [comment]
Okay, this has been updated for the CLI default change in #81.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -5,49 +5,61 @@ on:
...
+        if: github.event_name == 'pull_request' && matrix.lint
         uses: actions-rs/cargo@v1
         with:
-          command: test
-          args: --workspace --all-features
+          command: fmt
+          args: --all -- --check
+
+      - name: clippy
+        if: github.event_name == 'pull_request' && matrix.lint
```
Comment (cecton) [inline .github/workflows/main.yml:61]: nice!

**Review by cecton — APPROVED** [review]
Thanks a lot for your work. It's so top-notch

**Pull request merged**